app/controllers/statistics_controller.rb

```diff
@@ -8,6 +8,12 @@ class StatisticsController < Spotlight::ApplicationController
   before_action :attach_breadcrumbs
   before_action do
     authorize!(:read, current_exhibit)
+
```

Comment:
I toyed around with explicitly setting (infinite) limits for the fields we care about instead. I could go either way.